README.md

```diff
@@ -5,13 +5,25 @@ Chat bot for StackOverflow. Uses [amphp](https://github.com/amphp) libraries for
 
 [![Build Status](https://travis-ci.org/Room-11/Jeeves.svg?branch=master)](https://travis-ci.org/Room-11/Jeeves)
 
+## Requirements
```

Comment:
They are. It doesn't hurt to put it in the readme I suppose, but it does mean there are two places to update if deps are added/removed.